test/test_job.js

```diff
@@ -56,6 +56,13 @@ describe('Job', function(){
         expect(storedJob.opts.testOpt).to.be('enabled');
       });
     });
+
+    it('should use the custom jobId if one is provided', function() {
```

Comment:
Test case suggestion:

Create a job with a customId and test if it can be processed, in the `process` callback, verify that the custom jobId matches the one provided